build.sbt

```diff
@@ -43,6 +43,9 @@ val commonJvmSettings = commonSettings ++ Seq(
 )
 
 val commonJsSettings = commonSettings ++ Seq(
+  scalaJSLinkerConfig ~= {
+    _.withBatchMode(true).withParallel(false)
```

Comment:
maybe add (directly to master) a comment explaining why we need this and how does this fix the issue?

Reply:
I will add it